.github/workflows/upgrade_test.yaml

```diff
@@ -0,0 +1,55 @@
+name: Upgrade functionality test
+
+on: 
```

Comment:
Please change this to

```
on:
  workflow_dispatch:
```

We would merge this workflow with master. Currently we don't want to run it over every commit so we can run it manually. Once we are sure we can enable for push to `master` branch.

Once this change is done, we are good to merge.

Reply:
@sanketsudake Added workflow dispatch.